samples/snippets/deletes/deletes_snippets.py

```diff
@@ -0,0 +1,167 @@
+#!/usr/bin/env python
+
+# Copyright 2022, Google LLC
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# [START bigtable_deletes_print]
+from google.cloud import bigtable
+
+# Write your code here.
+# [START_EXCLUDE]
```

Comment:
why are we adding excludes?

Reply:
I copied it from the filter samples. I have removed the exclude tag.